CHANGES.md

```diff
@@ -69,6 +69,8 @@
 
 <!-- Changes to how Black is packaged, such as dependency requirements -->
 
+- Hatchling is now used as the build backend. (#3233)
```

Comment:
Maybe add something like "This is not expected to have any effects for users who install Black from PyPI." (That's true, right?)

Reply:
Done, and yes!